stable/velero/README.md

```diff
@@ -1,9 +1,15 @@
 # Velero-server
 
-This helm chart installs Velero version v0.11.0
-https://github.com/heptio/velero/tree/v0.11.0
+This helm chart installs Velero version v1.0.0
+https://github.com/heptio/velero/tree/v1.0.0
 
 
+## Upgrading to v1.0.0
+
+As of v1.0.0, Heptio Velero is no more backwards-compatible with Heptio Ark.
+
+The [instructions found here](https://heptio.github.io/velero/v1.0.0/upgrade-to-1.0) will assist you in upgrading from version v0.11.0 to v1.0.0
```

Comment:
We need to do a more detailed sweep of these docs, but for now the 1.0 docs live at https://velero.io/docs/v1.0.0/upgrade-to-1.0/

Reply:
I've corrected all the links